src/core/openthread-core-default-config.h

```diff
@@ -1535,4 +1535,34 @@
 #define OPENTHREAD_CONFIG_DISABLE_CCA_ON_LAST_ATTEMPT 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE
+ *
+ * Define openthread diagnostic mode output buffer size
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE
+#define OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE (256)
```

Comment:
while parentheses are typically a good idea in macros, this does not coincide with the style of the file.  Remove here and elsewhere below.

Reply:
Okay.